acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/TestDataSteps.java

```diff
@@ -78,11 +78,16 @@ public void discardEditedDocument() throws Throwable {
      */
     @After(value = "@TakeOfflineAfter", order = 500)
     public void takePublicationOffline() throws Throwable {
-        final String currentPublicationName = testDataRepo.getCurrentPublication().getName();
-        log.debug("Taking current publication offline: {}.", currentPublicationName);
+        // Don't want this to fail the test if it fails, it's just clean up
+        try {
+            final String currentPublicationName = testDataRepo.getCurrentPublication().getName();
+            log.debug("Taking current publication offline: {}.", currentPublicationName);
 
-        contentPage.openCms();
-        contentPage.openContentTab();
-        contentPage.unpublishDocument(currentPublicationName);
+            contentPage.openCms();
+            contentPage.openContentTab();
+            contentPage.unpublishDocument(currentPublicationName);
+        } catch (Exception e) {
+            log.error("Failed to take publication offline.", e);
+        }
```

Comment:
Good catch! :+1: Admittedly, I do wonder if there is a risk that this will make it less obvious if subsequent search tests fail because the cleanup didn't happen. I guess now that our tests of the published/upcoming page are more fuzzy, the risk of that is low. Plus, you do log the error after all so it's not like we're completely without the feedback.

Pun intended.

I'll show myself out.